core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -509,6 +516,21 @@ public boolean shutdownWorkerGroupOnStop() {
         return shutdownWorkerGroupOnStop;
     }
 
+    /**
+     * Returns the worker {@link EventLoopGroup} which is responsible for running
+     * {@link Service#serve(ServiceRequestContext, Request)}.
+     */
+    public EventLoopGroup serviceWorkerGroup() {
```

Comment:
Ditto - We should remove these methods. A user will get them from `ServiceConfig.serviceWorkerGroup()` or `VirtualHost.serviceWorkerGroup()`.